packages/tc-schema-sdk/data-accessors/relationship-types.js

```diff
@@ -0,0 +1,32 @@
+const type = require('./type');
+
+module.exports = {
+	// todo move to object rest parameters for options when upgrading node
+	accessor(options = {}) {
+		const types = this.rawData.getTypes();
+		const relationshipTypes = this.rawData.getRelationshipTypes();
```

Comment:
```suggestion
		const userDefinedRelationshipTypes = this.rawData.getRelationshipTypes();
```
This makes it a bit easier to follow I think